ts/src/renderer/phaser/classes/PhaserProjectile.ts

```diff
@@ -22,27 +24,32 @@ class PhaserProjectile extends Phaser.GameObjects.Container {
 				console.log('PhaserProjectile play-animation', `${key}/${animationId}`);  // TODO remove
 				sprite.play(`${key}/${animationId}`);
 			});
+
+		//console.log('projectile', projectile._scale, projectile._pixiTexture.scale, projectile._stats.currentBody.width)
 	}
 
 	update (/*time: number, delta: number*/): void {
 
 		const projectile = this.projectile;
-		const container = projectile._pixiContainer;
-		const texture = projectile._pixiTexture;
+		//const container = projectile._pixiContainer;
+		//const texture = projectile._pixiTexture;
 
-		if (projectile._destroyed || container._destroyed) {
+		if (!projectile._alive/*projectile._destroyed || container._destroyed*/) {
+			console.log('projectile destroy', projectile)
 			projectile.off('play-animation', this.playAnimationListener);
 			this.playAnimationListener = null;
-
+			this.scene.events.off('update', this.update, this);
 			this.destroy();
 			return;
 		}
 
-		this.x = container.x;
-		this.y = container.y;
+		this.x = projectile._translate.x;
+		this.y = projectile._translate.y;
+		//this.x = container.x;
+		//this.y = container.y;
 
 		const sprite = this.sprite;
-		sprite.rotation = texture.rotation;
-		sprite.setScale(texture.scale.x, texture.scale.y);
+		sprite.rotation = projectile._rotate.z; //texture.rotation;
+		//sprite.setScale (projectile._scale.x, projectile._scale.y); //texture.scale.x, texture.scale.y);
```

Comment:
can a projectile change scale compared to the initial set at creation time? if so, we should update it accordingly here